app/src/main/java/com/codesoom/assignment/repository/TaskRepository.java

```diff
@@ -0,0 +1,88 @@
+package com.codesoom.assignment.repository;
+
+import com.codesoom.assignment.models.Task;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Task Entity의 데이터를 보관하고 이 Entity의 CRUD와 관련된 로직을 담당합니다.
+ */
+@SpringBootApplication(scanBasePackages={"com.codesoom.assignment.repository"})
+public class TaskRepository {
+    private List<Task> tasks = new ArrayList<>();
+    private Long newId = 0L;
+
+    /**
+     * 저장된 할 일들(tasks)을 모두 반환합니다.
+     *
+     * @return 모든 할 일들(tasks)
+     */
+    public List<Task> findAll(){
+        return tasks;
+    }
+
+    /**
+     * 새로운 할 일(task)을 생성합니다.
+     *
+     * @param : 할 일(task)의 내용
+     * @return : 저장된 할 일(task)을 반환합니다.
+     */
```

Comment:
이 주석에는 문제가 있습니다. "새로운 할 일을 생성"한다고 되어 있는데 실제로 하는 일은 생성이 아니에요. 주어진 할 일 객체에 아이디를 부여하고, 저장소에 등록하는 일을 하고 있습니다. 하는 일을 보면 이 메소드의 올바른 이름은 `register` 또는 `save`가 더 맞을지도 모릅니다.

한편 javadoc 포맷은 다음과 같이 작성하도록 연습해 봅시다.

```suggestion
    /**
     * 새로운 할 일을 생성해 리턴합니다.
     *
     * @param task 새로 생성할 할 일
     * @return 생성된 할 일 인스턴스
     */
```

`@param`은 parameter의 이름을 적어줘야 합니다. `:`은 표준 포맷이 아니에요. `@return`에서는 어떤 것을 리턴하는지를 명사형으로 짧게 적어줍니다.